documentation/faq/80-integrations.md

```diff
@@ -50,6 +50,17 @@ onMount(() => {
 });
 ```
 
+Otherwise, if the library has side effects and you'd still prefer to use static imports, check out [vite-plugin-iso-import](https://github.com/bluwy/vite-plugin-iso-import) to support the `?client` import suffix. The import will be stripped out in SSR builds.
```

Comment:
It would probably make sense to add a sentence to the end of this paragraph like:
>However, note that you will lose the ability to use VS Code Intellisense if you use this method.

Reply:
Sounds good to clear that upfront. Added.